.github/workflows/cherryPick.yml

```diff
@@ -26,9 +26,28 @@ jobs:
 
   createNewVersion:
     needs: validateActor
+    runs-on: ubuntu-latest
     if: ${{ fromJSON(needs.validateActor.outputs.IS_DEPLOYER) && github.event.inputs.NEW_VERSION == '' }}
-    uses: Expensify/App/.github/workflows/createNewVersion.yml@main
-    secrets: inherit
+    outputs:
+      NEW_VERSION: ${{ steps.getNewVersion.outputs.NEW_VERSION }}
+    steps:
+      - name: Checkout main branch
+        uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f
+        with:
+          ref: main
```

Comment:
we no longer need `token: ${{ secrets.OS_BOTIFY_TOKEN }}` for this step?

Reply:
Nope. Never did 🙂